### PR TITLE
Towner Spam

### PR DESCRIPTION
**Pull request by Stagrawl**
Adds two new towners, the Minstrel + Drunkard.
Fixes Seamstress starting inventory.
Gives acolytes +1 medicine for sanity reasons.
Reduces standard beggar stealing by 1, because what the fuck.
Gives the physician his drip back??? wtf?

### CODE REVIEW

**Comment by assortedbeads** [comment]
Drunkard has knives 1 twice

**Comment by WindowTalents** [comment]
We already have beggar, the best role in the game, which you want to nerf because it is apparently too powerful..? What is this powercreep role Drunkard? I can literally just roleplay this as anybody? This is just free stats. Towner bard is, fine, but pointless.

Add this stuff TO BEGGAR - why do you want a buffer beggar on the towner slot? Make this a beggar variant please.

I don't want this to seem rude, I do not understand the purpose of these towners. (I just tried to make a very niche towner myself.)

**Comment by Stagrawl** [comment]
> We already have beggar, the best role in the game, which you want to nerf because it is apparently too powerful..? What is this powercreep role Drunkard? I can literally just roleplay this as anybody? This is just free stats. Towner bard is, fine, but pointless.
> 
> Add this stuff TO BEGGAR - why do you want a buffer beggar on the towner slot? Make this a beggar variant please.
> 
> I don't want this to seem rude, I do not understand the purpose of these towners. (I just tried to make a very niche towner myself.)

Drunkard is a different gimmick to the beggar.

Also. beggars shouldn't be master fucking thieves rivalling rogues and shit, they're **beggars**, it's fucking dumb and the justification for it is bad.

**Comment by Real-MAGNUM** [comment]
Fuck beggars. They're a pop overfill role at their core design, not a master thief.

**Comment by WindowTalents** [comment]
I am not against taking a skill level off beggars pickpocketing, I just - do not get why a drunkard gets to be a towner lol.

I thought towners at least received some sort of payment or reward, for their "job" that they do for the town. At least indirectly in the setting.

It just feels like, that's a beggar - but with str and con buffs. I run around as a drunk old beggar, that's literally my gimmick lol.

**Comment by Stagrawl** [comment]
> I am not against taking a skill level off beggars pickpocketing, I just - do not get why a drunkard gets to be a towner lol.
> 
> I thought towners at least received some sort of payment or reward, for their "job" that they do for the town. At least indirectly in the setting.
> 
> It just feels like, that's a beggar - but with str and con buffs. I run around as a drunk old beggar, that's literally my gimmick lol.

My intention is that they're a sort of gambler role who takes up space in the bar and provides environment and interacts with the game a little more meaningfully than what is, essentially, as pointed out above, a pop overfill solution. 

The original concept was "gambler" but shifted to this, which I'm not really against. It's been said that we want more roles that are just like "town drunk/village idiot" that aren't just a horde of nameless beggartide. 

Current focus on expanding town slop roles too, so it's more of a "spin the wheel and see what you get!" not just "pick from this curated list" type deal.

**Comment by Stagrawl** [comment]
If you (or anyone else) have some other towner roles you'd like to see that fulfil some niche and add something to the environment, I'd love to hear it.

**Comment by WindowTalents** [comment]
Please leave it as a curated list.
I genuinely hate being told what to play by that kind of stuff.

Just add more towner variety. Just don't give them stupid good stats like this, lol.

**Comment by TheManWithNoHands** [comment]
> Gives the physician his drip back??? wtf?

Wupps, i fucked up in solving merge conflicts it seems. Weird. Sorry about that.

File: code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/drunkard.dm [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/knives, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/craft/alchemy, 1, TRUE) //They know what substances is and things.
+		H.mind.adjust_skillrank(/datum/skill/misc/lockpicking, 1, TRUE) //Breaking into the houses of anyone who ever loved them.
+		H.mind.adjust_skillrank(/datum/skill/misc/climbing, 1, TRUE) //Climbing into windows to steal drugs or booze.
+		H.change_stat("intelligence", -2)
+		H.change_stat("constitution", 2)
+		H.change_stat("strength", 2)
+		ADD_TRAIT(H, TRAIT_NOSTINK, TRAIT_GENERIC)
```
Comment (ChungusGamer666) [inline code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/drunkard.dm:48]: This feels random

Comment (Stagrawl) [reply code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/drunkard.dm:48]: Constitution from a lifetime of hearty drinking without dying, strength from the rage of a man on his fourth beer of the hour, or someone who wants your drugs. Lack of intelligence from a lifetime of bad decisions.

Comment (ChungusGamer666) [reply code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/drunkard.dm:48]: Nostink?

Comment (Stagrawl) [reply code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/drunkard.dm:48]: No olefactory function at all. All synapses burned away long ago. Also, has own drunken stench which overpowers even the most intense of any miasma.

I did put a tiny bit of thought into this, have some faith :(

**Comment by CVSthePharmacy** [comment]
role inflation, also drunkard role when we already have drunkard beggars.... why??? I don't like this pr for that reason, I feel like it should be closed due to the role slop

**Comment by CVSthePharmacy** [comment]
or at least closed and made into a different pr without drunkard

**Comment by Stagrawl** [comment]
> role inflation, also drunkard role when we already have drunkard beggars.... why??? I don't like this pr for that reason, I feel like it should be closed due to the role slop

Role inflation isn’t a thing on advclasses, also I’ll inflate you. 

I’ll just rename it to Gambler and tweak it a bit.

**Comment by CVSthePharmacy** [comment]
> > role inflation, also drunkard role when we already have drunkard beggars.... why??? I don't like this pr for that reason, I feel like it should be closed due to the role slop
> 
> Role inflation isn’t a thing on advclasses, also I’ll inflate you.
> 
> I’ll just rename it to Gambler and tweak it a bit.

if you do that then just increase fortune and remove intelligence debuff.

**Comment by WindowTalents** [comment]
My issue is still that gamblers do not provide for the town. This should be some sort of "games master" that wants to encourage other people TO gamble - not the other way around.

The House Always Wins, afterall...

I think minstrel is a neat idea, but the "gambler" makes no sense to me. This is just a towner beggar, I don't get it. It needs some sort of unique "thing" it "provides" for the town. A towner bard is fine, but some guy who drinks and gambles as his career - is a bum lol.

I would honestly take that part of this off your PR, and focus on the Minstrel first, but that is just my opinion.
You changed a ton of shit with this PR instead of focusing on the new roles, that's basically what I'm having a problem with. I'm also just a player on the server, so my opinion is merely that.

I will absolutely use Minstrel if it is added - but I already roleplay beggar as the other role haha.

**Pull request merged**